docs/src/app/(public)/(content)/react/components/toast/demos/position/css-modules/index.module.css

```diff
@@ -122,6 +127,19 @@
   }
 }
 
+.Content {
+  overflow: hidden;
+  transition: opacity 0.25s;
+
+  &[data-behind] {
+    opacity: 0;
+  }
```

Comment:
Would this have the same effect?


```diff
-  &[data-behind] {
     opacity: 0;
-  }
```

Am I right at assuming that a toast is always in the "behind" or "expanded" state, and only in one state at a time?